application/app/process/script_launcher.rb

```diff
@@ -102,4 +102,10 @@ def update_lock_file(file, pid)
     log_error("Failed to update lock file", { pid: pid, error: e.message })
     raise
   end
+
+  def script_log_filename
+    # ISO week number with year, e.g. launch_detached_process-2025-W38.log
+    year, week = Date.today.cweek, Date.today.cwyear
+    "launch_detached_process-#{year}-W#{format('%02d', week)}.log"
```

Comment:
shouldn't this be the other way around? In my local setup and in the test the log file name is being `launch_detached_process-39-W2025.log` instead of `launch_detached_process-2025-W39.log`

```suggestion
    week, year = Date.today.cweek, Date.today.cwyear
    "launch_detached_process-#{year}-W#{format('%02d', week)}.log"
```